### ISSUE TITLE
Black screen in v0.9 in Windows 11

### ISSUE THREAD

**Issue by CodeSpartan** [mausimus/ShaderGlass#87]
Black screen in v0.9 in Windows 11
Not sure if it helps, but I built it in Debug Mode to see if anything pops up and I had this message spam in the output log:
`D3D11 ERROR: ID3D11DeviceContext::Draw: Vertex Shader - Pixel Shader linkage error: Signatures between stages are incompatible. The input stage requires Semantic/Index (TEXCOORD,1) as input, but it is not provided by the output stage. [ EXECUTION ERROR #342: DEVICE_SHADER_LINKAGE_SEMANTICNAME_NOT_FOUND]`

**Comment by CodeSpartan** [comment]
No, the problem was due to two monitors. The default option in Input -> Desktop has "All Displays" selected, and it took input at coordinates that had nothing - so it displayed black screen, no matter how much you moved the window. Just select your desired monitor and it's fixed.

**Comment by mausimus** [comment]
Hey, do you still have this issue when starting up on two monitors? If yes, is there anything special about your setup, like different refresh rates, or unusual monitor layout? Dual monitor should generally work fine by default.

BTW that debug warning is due to a minor issue with shader code, it doesn't affect SG operation.
